build_tools/buildkite/cmake/linux/x86_64/benchmark2.yml

```diff
@@ -27,4 +27,4 @@ steps:
       - "queue=benchmark-x86_64"
     artifact_paths:
       - "benchmark-results-gcp-cpu-${BUILDKITE_BUILD_NUMBER}.json"
-    timeout_in_minutes: "20"
+    timeout_in_minutes: "30"
```

Comment:
The benchmark time is growing... We will probably need to solve this in the new benchmark pipeline by splitting the workload.

Reply:
Yeah I was worried about this :-/